browser/brave_content_browser_client.cc

```diff
@@ -190,6 +190,11 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/browser/brave_ads/brave_ads_host.h"
 #endif  // defined(OS_ANDROID)
 
+#if !defined(OS_ANDROID)
+#include "brave/browser/ui/webui/brave_shields/shields_panel_ui.h"
```

Comment:
Let's use above `#if !defined(OS_ANDROID)` section instead of adding same define.